RELEASES.md

```diff
@@ -27,6 +27,7 @@ Libraries
 - [Extend `ptr::null` and `null_mut` to all thin (including extern) types.][94954]
 - [`impl Read and Write for VecDeque<u8>`.][95632]
 - [STD support for the Nintendo 3DS.][95897]
+- [Use rounding in float to Duration conversion methods.][96051]
```

Comment:
Can we also add this to the future compat section below?

Reply:
@Mark-Simulacrum is this what you meant?
https://github.com/rust-lang/rust/pull/100482/commits/cee573f32a1927ba938a29343bab2121136c5b28